installer/templates/phx_assets/package.json

```diff
@@ -10,21 +8,20 @@
     "phoenix": "file:<%= phoenix_webpack_path %>"<%= if html do %>,
     "phoenix_html": "file:<%= phoenix_html_webpack_path %>"<% end %><%= if live do %>,
     "phoenix_live_view": "file:<%= phoenix_live_view_webpack_path %>",
-    "nprogress": "^0.2.0"<% end %>
+    "nprogress": "^0.2"<% end %>
```

Comment:
Minor nitpick but something I noticed when you first install something with `npm` it will reorder this to put it above phoenix to respect alphabetical ordering of the dependencies. Perhaps we can move it there now and save future projects from that diff?

Reply:
I agree. Also concerns `npm update` it will always reorder it.